packages/next/client/performance-relayer.ts

```diff
@@ -46,7 +46,7 @@ function onReport(metric: Metric) {
       // This content type is necessary for `sendBeacon`:
       type: 'application/x-www-form-urlencoded',
     })
-    const vitalsUrl = 'https://vitals.vercel-analytics.com/v1/vitals'
+    const vitalsUrl = 'https://vitals.vercel-insights.com/v1/vitals'
     ;(navigator.sendBeacon && navigator.sendBeacon(vitalsUrl, blob)) ||
```

Comment:
That semi-colon at the beginning 👀

Reply:
Intentional to prevent a function call against the string!